_data/contents.yml

```diff
@@ -39,4 +39,5 @@ specifics:
   - debugging
 libraries:
   - guardian
+  - guardian.0.14.x
```

Comment:
@hassox how do you feel about removing this lesson all together?  Let's focus only on the latest and greatest 😁

Reply:
I'm have no opinion either way ;) Happy to remove.

Reply:
Let's just replace it 👍